python/triton/compiler/compiler.py

```diff
@@ -45,7 +45,12 @@ def ttir_compute_capability_rewrite(mod, arch):
     pm = ir.pass_manager(mod.context)
     pm.enable_debug()
     if _is_cuda(arch):
-        pm.add_rewrite_tensor_pointer_pass(arch)
+        pm.add_rewrite_tensor_pointer_pass(arch, False)
+    else:
+        device = get_current_device()
```

Comment:
This line failing in offline tests, because on github ci we do not have any rocm capable devices.